File: Temp/leetcode/editor/cn/[2390]从字符串中移除星号.c
// 2390 从字符串中移除星号
// https://leetcode.cn/problems/removing-stars-from-a-string/


//leetcode submit region begin(Prohibit modification and deletion)
char* removeStars(char* s) {
    int i = 0, j = 0;
    for (; s[i]; i++) {
        if (s[i] == '*')
            j--;
        else
            s[j++] = s[i];
    }
    s[j] = '\0';
    return s;
}
//leetcode submit region end(Prohibit modification and deletion)

File: Temp/leetcode/editor/cn/[2390]从字符串中移除星号.py
# 2390 从字符串中移除星号
# https://leetcode.cn/problems/removing-stars-from-a-string/


# leetcode submit region begin(Prohibit modification and deletion)
class Solution:
    def removeStars(self, s: str) -> str:
        st = []
        for c in s:
            if c == "*":
                st.pop()
            else:
                st.append(c)
        return "".join(st)
# leetcode submit region end(Prohibit modification and deletion)
